tests/lib/rules/jsx-props-no-spreading.js

```diff
@@ -27,170 +27,235 @@ const ruleTester = new RuleTester({parserOptions});
 const expectedError = {message: 'Prop spreading is forbidden'};
 
 ruleTester.run('jsx-props-no-spreading', rule, {
-  valid: [{
-    code: [
-      'const {one_prop, two_prop} = props;',
-      '<App one_prop={one_prop} two_prop={two_prop}/>'
-    ].join('\n')
-  }, {
-    code: [
-      'const {one_prop, two_prop} = props;',
-      '<div one_prop={one_prop} two_prop={two_prop}></div>'
-    ].join('\n')
-  }, {
-    code: [
-      'const newProps = {...props};',
-      '<App one_prop={newProps.one_prop} two_prop={newProps.two_prop} style={{...styles}}/>'
-    ].join('\n')
-  }, {
-    code: [
-      'const props = {src: "dummy.jpg", alt: "dummy"};',
-      '<App>',
-      '   <Image {...props}/>',
-      '   <img {...props}/>',
-      '</App>'
-    ].join('\n'),
-    options: [{exceptions: ['Image', 'img']}]
-  }, {
-    code: [
-      'const props = {src: "dummy.jpg", alt: "dummy"};',
-      'const { src, alt } = props;',
-      '<App>',
-      '   <Image {...props}/>',
-      '   <img src={src} alt={alt}/>',
-      '</App>'
-    ].join('\n'),
-    options: [{custom: 'ignore'}]
-  }, {
-    code: [
-      'const props = {src: "dummy.jpg", alt: "dummy"};',
-      'const { src, alt } = props;',
-      '<App>',
-      '   <Image {...props}/>',
-      '   <img {...props}/>',
-      '</App>'
-    ].join('\n'),
-    options: [{custom: 'enforce', html: 'ignore', exceptions: ['Image']}]
-  }, {
-    code: [
-      'const props = {src: "dummy.jpg", alt: "dummy"};',
-      'const { src, alt } = props;',
-      '<App>',
-      '   <img {...props}/>',
-      '   <Image src={src} alt={alt}/>',
-      '   <div {...someOtherProps}/>',
-      '</App>'
-    ].join('\n'),
-    options: [{html: 'ignore'}]
-  }, {
-    code: `
+  valid: [
```

Comment:
there's some more reformatting here; can that be reverted?

Reply:
Done